### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by Satyajit1993** [AarohiSingla/YOLO-NAS#1]
While running 

trainer.train(model=model, 
              training_params=train_params, 
              train_loader=train_data, 
              valid_loader=val_data)

this command I am getting an error as

'charmap' codec can't encode characters in position character maps to <undefined>

**Comment by AarohiSingla** [comment]
This type of error commonly occurs when there are non-ASCII characters in the data or the code file.
In Python, the recommended encoding is UTF-8. Check the encoding of your code file and ensure it matches.

**Comment by Louis-Dupont** [comment]
@Satyajit1993 is it the exact same error as in this issue https://github.com/Deci-AI/super-gradients/issues/999 ?
If yes, feel free to copy/paste the full error to that issue, along with information about your environment (like the person who opened the issue), this would help us better understand and fix it!
